basic_datatypes.ipynb

```diff
@@ -321,8 +322,8 @@
     "In the cell below you should write the code that solves the following exercises:\n",
     "\n",
     "  1. Write a Python program to that gets three float numbers and retuns $(a + b) * c$.\n",
-    "  2. Write a Python program that computes the area of a circle with radius $r$.\n",
-    "  2. Write a Python program that returns a solution of quadratic equation $ax^2 + bx + c = 0$. We consider only the case when solutions are real numbers."
+    "  2. Write a Python program that computes the area of a circle with radius $r$. Use math.pi from the math library as value of pi.\n",
+    "  2. Write a Python program that returns the solutions of the quadratic equation $ax^2 + bx + c = 0$. We consider only the case when there are two solutions and both are real numbers."
```

Comment:
No the "two" solutions can be the same and then I wouldn't consider this valid, that's what I meant by two. 
E.g. I don't allow x^2 -1 = 0 because then it's unclear, should I return 1 or return (1,1)

Reply:
I will write distinct, i think that makes it clearer